pom.xml

```diff
@@ -191,7 +191,7 @@
                         <dependency>
                             <groupId>com.fasterxml.jackson.module</groupId>
                             <artifactId>jackson-module-kotlin</artifactId>
-                            <version>2.12.2</version>
+                            <version>2.12.3-SNAPSHOT</version>
```

Comment:
Setting this to the current snapshot because this changeset effectively removes a bunch of stuff from the API.